pkg/storage/unified/search/dashboard.go

```diff
@@ -211,8 +209,7 @@ func (s *DashboardDocumentBuilder) BuildDocument(ctx context.Context, key *resou
 	}
 
 	// Add the stats fields
-	stats := s.Stats(ctx, key.Name) // summary.UID
-	for k, v := range stats {
+	for k, v := range s.Stats[summary.UID] {
```

Comment:
TODO: Going to change how stats are keyed. Right now its by dashboard uid, but can't guarantee those are globally unique.

Reply:
actually nvm the dashboard builders are unique per namespace so its fine how it is now